semgrep-core/src/parsing/tree_sitter/Parse_csharp_tree_sitter.ml

```diff
@@ -3036,15 +3036,22 @@ let parse file =
         pr2 s;
         raise exn)
 
-let parse_pattern str =
```

Comment:
You can keep parse_pattern, because later it could be 
parse_expression_or_stmt_or_multistmts_or_compilation_unit